### PR TITLE
Add hindi translation

### PR DESCRIPTION
**Pull request by shellheim**
# Hey!  
This PR adds Hindi translation, addressing #883.
Thanks to @saghul for allowing me to contribute to jitsi!  
Also sorry for not making a draft PR at first.

### CODE REVIEW

File: app/i18n/index.js [hunk 1]
```diff
@@ -1,36 +1,35 @@
-import i18n from 'i18next';
-import { initReactI18next } from 'react-i18next';
-import moment from 'moment';
+import i18n from "i18next";
```
Comment (saghul) [inline app/i18n/index.js:1]: Please do t change the existing style

File: app/i18n/index.js [hunk 2]
```diff
@@ -1,36 +1,35 @@
...
+import { initReactI18next } from "react-i18next";
+import moment from "moment";
 
 const languages = {
-    de: { translation: require('./lang/de.json') },
```
Comment (saghul) [inline app/i18n/index.js:6]: Please don't change the existing style. Just add your changes with the same style as the surrounding code.

**Review by saghul — CHANGES_REQUESTED** [review]

File: app/i18n/index.js (hunk 1 above)
Comment (shellheim) [reply app/i18n/index.js:1]: I am so sorry, wasn't intentional. I was working with neovim and something about how it deals with copy/paste or something else entirely might have caused this, let me fix it.

**Comment by shellheim** [comment]
It would be way more simple for me to do it from scratch then actually try to figure out git right now.

**Pull request closed without merging**